tests/Tests/500__jupyterhub/test-jupyterlab-time.robot

```diff
@@ -0,0 +1,83 @@
+*** Settings ***
+Resource            ../../Resources/ODS.robot
+Resource            ../../Resources/Common.robot
+Resource            ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource            ../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+Library             ../../../../libs/Helpers.py
+Library             DateTime
+Library             OpenShiftCLI
+Library             DebugLibrary
+
+Suite Setup         Load Spawner Page
+Suite Teardown      End Web Test
+
+
+*** Variables ***
+@{LIST_OF_IMAGES}       s2i-minimal-notebook    s2i-generic-data-science-notebook
+...                     pytorch                 tensorflow    minimal-gpu
+
+${LIMIT_TIME} =    40
+
+*** Test Cases ***
+Average Time For Spawning
+    [Documentation]    Verifies that average spawn time for all JupyterHub images is less than 40 seconds
+    [Tags]    ODS-691
+    ...       Tier2
+    ${total_avg} =    Set Variable
+    FOR    ${image}    IN    @{LIST_OF_IMAGES}
+        ${avg} =    Set Variable
+        Spawn and Stop Server    ${image}
+        FOR    ${counter}    IN RANGE    4
+            Close Previous Tabs
+            ${sum} =    Spawn and Stop Server    ${image}
+            ${avg} =    Evaluate    ${avg} + ${sum}
+        END
+        ${avg} =    Evaluate    ${avg}/4
+        Log    ${avg}
+        ${total_avg} =    Evaluate    ${total_avg} + ${avg}
+
+    END
+    ${len} =    Get Length    ${LIST_OF_IMAGES}
+    ${total_avg} =    Evaluate    ${total_avg} / ${len}
+    Log    total_avg time to spawn ${total_avg}
+    ${result} =    lt    ${total_avg}.0    ${LIMIT_TIME}.0.0
+    Run Keyword Unless    ${result}    Fail
```

Comment:
could you please re-write this TC following the style proposed by [ODS-CI Style Guide](https://docs.google.com/document/d/11ZJOPI1uq-0Wl6a2V8fkAv_TQhfzp9t_IjXAheaJxmQ/)

Reply:
Done